### PR TITLE
Profile/edit user

### PR DESCRIPTION
**Pull request by hawkprovvv**

### CODE REVIEW

File: app/api/margatsni/v1/endpoints/users.rb [hunk 1]
```diff
@@ -36,6 +40,26 @@ def represent_user_with_token(user)
...
+
+          namespace :me do
+            desc 'profile'
+            get do
+              represent_current_user(current_user)
+            end
+
+            desc 'edit user'
+            params do
+              requires :username, type: String, allow_blank: false
```
Comment (slenderock) [inline app/api/margatsni/v1/endpoints/users.rb:52]: params for editing couldn't be required

File: app/api/margatsni/v1/endpoints/users.rb [hunk 2]
```diff
@@ -36,6 +40,26 @@ def represent_user_with_token(user)
...
+              requires :email, type: String, regexp: User::EMAIL_REGEXP
+              requires :password, type: String, allow_blank: false
+              requires :bio, type: String, allow_blank: true
+            end
+            put do
+              current_user.update(declared(params))
```
Comment (slenderock) [inline app/api/margatsni/v1/endpoints/users.rb:58]: add `include_missing: false`

File: test/fixtures/administrators.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+one:
```
Comment (slenderock) [inline test/fixtures/administrators.yml]: delete the file which is part of incompleted tests

File: test/models/administrator_test.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AdministratorTest < ActiveSupport::TestCase
```
Comment (slenderock) [inline test/models/administrator_test.rb]: delete the file which is part of incompleted tests

**Review by slenderock — CHANGES_REQUESTED** [review]

File: app/api/margatsni/v1/endpoints/users.rb (hunk 2 above)
Comment (hawkprovvv) [reply app/api/margatsni/v1/endpoints/users.rb:58]: We have a 'bio' parameter, and it does not necessarily have value. For other parameters, we have a validator 'allow_blank: false'

**Review by slenderock — APPROVED** [review]

**Pull request merged**